tests/10-issue.js

```diff
@@ -26,95 +26,284 @@ describe('BitstringStatusList Credentials (Issue "statusPurpose: revocation")',
         let issuerResponse;
         let err;
         let issuedVc;
+        beforeEach(function() {
+          this.test.cell = {
+            columnId: issuerName, rowId: this.test.title
+          };
+        });
         before(async function() {
-          const issuer = issuers.find(issuer => issuer.tags.has('Revocation'));
+          const issuer = issuers.find(
+            issuer => issuer.tags.has('BitstringStatusList'));
           const {result, error, data} = await issueVc({issuer});
           err = error;
           issuerResponse = result;
           issuedVc = data;
+          const credentialStatusType = typeof (issuedVc.credentialStatus);
+          if(credentialStatusType === 'object') {
+            // TODO make everything an array and test each status entry
+            // issuedVc.credentialStatus = [issuedVc.credentialStatus]
+          }
         });
-        it('MUST issue a VC with BitstringStatusList "credentialStatus" and ' +
-          '"revocation" status purpose', async function() {
-          this.test.cell = {columnId: issuerName, rowId: this.test.title};
-          should.exist(issuerResponse);
-          should.not.exist(err);
-          issuerResponse.status.should.equal(201);
-          should.exist(issuedVc, `Expected VC from ${issuerName} to exist.`);
-          testCredential({credential: issuedVc});
-        });
-        // ensure that issued StatusList Credential contain correct properties
-        it('MUST have correct properties when dereferencing ' +
-          '"credentialStatus.statusListCredential"', async function() {
-          this.test.cell = {columnId: issuerName, rowId: this.test.title};
-          const {slc} = await getSlc({issuedVc});
-          testSlCredential({slCredential: slc});
+        describe('BitstringStatusList Entry', function() {
+          it('Any expression of the data model in this section ' +
+            'MUST be expressed in a conforming verifiable credential ' +
+            'as defined in [VC-DATA-MODEL-2.0].',
```

Comment:
I reverted on that decision as it introduces the spacing in the newline.

Reply:
So the text above would look like:
`Any expression of the data model in this section             MUST be expressed in a conforming verifiable credential             as defined in [VC-DATA-MODEL-2.0].`

Unless we make it like so in the code (removing the spacing on the next line):
```
          it(`Any expression of the data model in this section \
MUST be expressed in a conforming verifiable credential \
as defined in [VC-DATA-MODEL-2.0].`,
```

If we are okay with this we can use it otherwise it makes readability difficult

Reply:
Yeah...lets just avoid it. I did wonder about that, but was thinking only of the HTML reports...where whitespace doesn't matter. Ah well.